source/set_up_3dsecure/index.html.md.erb

```diff
@@ -1,56 +1,70 @@
-4. Select __On__.
+You do not need to do anything because we set up 3DS2 on your service by default.
+
+Stripe automatically applies exemptions.
+
+## Set up SCA with Worldpay
+
+Worldpay sends all transactions using 3DS2 or 3DS1. 3DS Flex is the Worldpay product that enables 3DS2 payment authentication. You must pay a fee to Worldpay to use 3DS Flex.
+
+Worldpay sends you your 3DS Flex credentials when you [go live](/switching_to_live/set_up_a_live_worldpay_account/#connect-your-live-account-to-worldpay).
+
+Worldpay does not apply any exemptions. Contact GOV.UK Pay to apply for exemptions for your transactions.
 
 ### Set up 3DS2 (3DS Flex)
 
-You must pay a fee to Worldpay to use 3DS Flex.
+1. Select the account you want to set up on the __My services__ page in the GOV.UK Pay admin tool.
+1. In __Settings__, make sure that 3D Secure is __On__.
+1. In __Your PSP__, enter your 3DS Flex credentials.
 
-You'll need your 3DS Flex details to set up your account. Ask your Worldpay account manager if you do not have your details.
+<%= warning_text('If your credentials are wrong, you are not able to take payments.') %>
 
-1. Select the account you want to set up on the [__My services__](https://selfservice.payments.service.gov.uk/my-services) page in the GOV.UK Pay admin tool.
-2. In the settings, select the __View__ link under __3D Secure__.
-3. Select __On__.
-4. Copy your 3DS Flex details into the [PSP settings in your GOV.UK Pay account](https://selfservice.payments.service.gov.uk/your-psp).
-5. Check your account still works by [making a payment with a real payment card](/switching_to_live/#6-check-your-live-account-works).
+### Set up 3DS1
 
-## ePDQ
+1. Ask your Worldpay account manager to set up 3DS on your merchant code for all payments.
+1. Once Worldpay have set up your merchant code, select the account you want to set up on the __My services__ page in the GOV.UK Pay admin tool.
+1. In the settings, select the __View__ link under 3D Secure.
+1. Select __On__.
 
-1. Sign in to your [ePDQ account](https://payments.epdq.co.uk/Ncol/Prod/BackOffice/login/index).
-2. Go to __Configuration__ then  __Payment methods__.
-3. Select __3D Secure__.
-4. Check that 3DS is on.
-5. In the GOV.UK Pay admin tool, go to your [__My services__](https://selfservice.payments.service.gov.uk/my-services) page, then select the account you want to set up
-6. In the settings, select the __View__ link under __3D Secure__.
-7. Select __On__.
+## Set up SCA with ePDQ
 
-## SmartPay
+You do not need to do anything because we set up 3DS2 on your service by default.
 
-1. Select the account you want to set up on the [__My services__](https://selfservice.payments.service.gov.uk/my-services) page in the GOV.UK Pay admin tool.
-2. In the settings, select the __View__ link under __3D Secure__.
-3. Select __On__.
+ePDQ does not apply any exemptions.
+
+## Set up SCA with Smartpay
+
+Smartpay does not support 3DS2. You can only enable 3DS1.
```

Comment:
I think we have decided not to support 3DS2 for smartpay, rather than smartpay not supporting 3ds2. though i'm not 100% certain.
maybe - 3ds2 is not available for Smartpay.

Reply:
Correct, SmartPay does support 3DS2 but we haven’t implemented it.